docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.5'
+services:
+  mobilic-db:
+    image: postgres:12.4-alpine
+    container_name: mobilic-db
+    # restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: mobilic
+      POSTGRES_USER: mobilic
+      POSTGRES_PASSWORD: mobilic
+    volumes:
+      - mobilic-postgresql-data:/var/lib/postgresql
+
+  mobilic-db-test:
+    image: postgres:12.4-alpine
+    ports:
+      - "5433:5432"
```

Comment:
Note that db test is on port 5433. I didn't use env var for this and directly put it in db test configuration.